web/src/components/PoisMobile.tsx

```diff
@@ -65,6 +65,20 @@ const StyledIcon = styled(FontAwesomeIcon)<{ direction: string }>`
     `};
 `
 
+const GeoLocateContainer = styled.div<{ isLastSnapPoint: boolean }>`
+  position: absolute;
+  right: 10px;
+  top: -40px;
```

Comment:
i think you should use a const for the `40` and reuse it in the transform transform translate